client/document/utils/bilmur.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Internal dependencies
+ */
+import config from '@automattic/calypso-config';
+
+export function isBilmurEnabled() {
+	return config.isEnabled( 'bilmur-script' );
+}
+
+/**
+ * Get the cache-busted URL for the bilmur script.
+ */
+export function getBilmurUrl() {
+	const baseUrl = config( 'bilmur_url' );
+
+	if ( ! isBilmurEnabled || ! baseUrl ) {
```

Comment:
I'd bet you meant to invoke `isBilmurEnabled()` here.

Reply:
Oh no, you're right! Thank you, @tyxla! 👍

Reply:
Thankfully there's a redundant check that prevents any actual errors, but this should be fixed in case that code is ever refactored.